src/back/CrossTeamTestSuite/src/CrossTeamTestSuite/Endpoints/Converters/ValueConverters/ListValueConverter.cs

```diff
@@ -0,0 +1,20 @@
+using System.Text;
+
+namespace CrossTeamTestSuite.Endpoints.Converters.ValueConverters;
+
+public class ListValueConverter<T> : IValueConverter<List<T>>
```

Comment:
It won't really work if `T` is `string`, because you would need to encode that string to be compatible with `Url`